discovery/README.md

```diff
@@ -0,0 +1,56 @@
+# Discovery of PFIs
+
+Paritipating Financial Institutions (PFIs) will likely wish to make their presence known to the public.
+
+This part of the protocol shows how this can be done in a decentralized fashion. 
+
+## Service Endpoints
+
+For a PFI to be discoverable, it must publish a DID Document with a service endpoint of type "PFI" at a minimum (from serviceEndpoints section of DID Document):
+
+```json
+      {
+        id:'pfi',
+        type: 'PFI',
+        serviceEndpoint: 'https://pfi.mic4.com',
+      }
+```      
+
+### Optional currency pair declaration
+
+To help clients find the right PFI for their needs, a PFI may also declare what currency pairs they are able to offer:
+
+```json
+      {
+        id:'pfi',
+        'type': 'PFI',
+        'serviceEndpoint': 'https://pfi.mic4.com AUD-USD,AUD-TZN,USD-BTC',
+      },
+```
+
+This is a comma seperated list of source-target currency pairs which a client may choose to use to display following the serviceEndpoint. The PFI has a formal offer protocol but this can be used to accelerate the discovery process.
+
+# Examples
+
+To test out the example code included here, you will need to install the dependencies:
+
+```bash
+npm install @decentralized-identity/ion-tools
+```
+
+
+## Sample usage
+
+```node discover.js```
+
+This will provide a list of PFIs. This uses a did:ion type of `1669` (16=P, 6=F, 9=I) to find all PFIs (and check they have a PFI type serviceEndpoint)
```

Comment:
can you explain why type is 1669?

Reply:
P is 16th letter, F is 6th, I is 9th. Just a way to remember it as the "PFI" type

Reply:
sorry, i was asking why it has to be 1669 and not just `pfi` :D does it have to be a number?

Reply:
@jiyoontbd yes has to be a number no more than 4 digits.